W-18724060-Update-cloud-IDE-help-CP (#460)
mulesoft/docs-code-builder: modules/ROOT/pages/_partials/acb-start-cloud.adoc

====
To save your work, commit your project files to a source control management (SCM) system. See xref:anypoint-code-builder::start-scm.adoc[].

> If you are using an Anypoint Platform trial account that loses access to Anypoint Code Builder, the cloud IDE instances are deleted.

Reviewer: I'd remove this whole sentence. Trial accounts won't be supported permanently.

Author: OK! Thanks!

> Do not use the beta version of Anypoint Code Builder to develop APIs, integrations, or automations for production environments.

Reviewer: Recommend clarifying that it's the cloud IDE that's beta

====
To save your work, commit your project files to a source control management (SCM) system. See xref:anypoint-code-builder::start-scm.adoc[].

> Do not use the cloud IDE beta version of Anypoint Code Builder to develop APIs, integrations, or automations for production environments.

Reviewer: ```suggestion
Don't use the cloud IDE beta version of Anypoint Code Builder to develop APIs, integrations, or automations for production environments.
```

Author: OK! Thanks!
